src/app/core/tools/TranslateTool.ts

```diff
@@ -104,8 +104,9 @@ export const TranslateTool: Tool = (() => {
                         curPositions.map(p => Snap(p)):
                         curPositions;
 
+                    const snapping = input.isShiftKeyDown() ? false : true;
```

Comment:
This sounds unintuitive because generally holding shift would be snapping, let's call this `snapToConnections`